datasets/NRS-Monthly-Data-on-Births-and-Deaths-Registered-in-Scotland/info.json

```diff
@@ -1,7 +1,7 @@
 {
     "title": "Monthly Data on Births and Deaths Registered in Scotland",
-    "publisher": "National Records of  Scotland",
-    "description": "Monthly data on births and deaths registered in Scotland. Table 1 presents the numbers of births registered in Scotland, and each National Health Service (NHS) Board area, for every month since 1990. Table 2 presents the numbers of births registered in Scotland, and each council area, for every month since 1996. Table 3 presents the numbers of deaths registered in Scotland, and each National Health Service (NHS) Board area, for every month since 1990. Table 4 presents the numbers of deaths registered in Scotland, and each council area, for every month since 1996. Table 5 presents the numbers of deaths registered in Scotland, by certain causes of death, for every month since 2000.",
+    "publisher": "National Records of Scotland",
+    "description": "Monthly data on births and deaths registered in Scotland. Table 1 presents the numbers of births registered in Scotland, and each National Health Service (NHS) Board area, for every month since 1990. Table 2 presents the numbers of births registered in Scotland, and each council area, for every month since 1996. Table 3 presents the numbers of deaths registered in Scotland, and each National Health Service (NHS) Board area, for every month since 1990. Table 4 presents the numbers of deaths registered in Scotland, and each council area, for every month since 1996. Table 5 presents the numbers of deaths registered in Scotland, by certain causes of death, for every month since 2000. The figures shown for the latest year (or two) will be provisional, pending the publication of the final figures for each calendar year in the following summer (for example the final figures for 2007 were published in summer 2008). Please note that there may be some minor discrepancies between the figures which are given here and those that are appear in the Quarterly and Preliminary Annual tables, because the tables may have been extracted at different times, and a small number of records may have been added to the statistical database in the intervening period. The full breakdown of cause of death is not shown because there is not enough time between registration and publication of monthly death figures to quality assure the data to provide reliable figures for more detailed causes of death. The causes in this dataset were chosen because they each account for more than 10 per cent of all deaths.",
```

Comment:
The description here is pretty large. I found it difficult to find the right place to put some of the footnotes from the import spreadsheets. It would be convenient to be able to create sub-sets of data and appy footnotes to them. Failing that, it would be easier if we could simply list a number of distinct comments against the dataset rather than having to mash it all in to one string here.